src/bot_commands.c

```diff
@@ -1651,7 +1675,8 @@ static frogbot_cmd_t std_commands[] = {
 	{ "removebot", FrogbotsRemovebot_f, "Removes a single bot" },
 	{ "removeall", FrogbotsRemoveAll, "Removes all bots from server" },
 	{ "debug", FrogbotsDebug, "Debugging commands" },
-	{ "disable", FrogbotsDisable, "Disable frogbots" }
+	{ "disable", FrogbotsDisable, "Disable frogbots" },
+	{ "dump", FrogbotsDumpSkills, "Dump skill CVARS to custom files" },
```

Comment:
since this writes to the server file system it doesn't seem like something anyone logged in should be able to do